### ISSUE TITLE
The button is not visible.

### ISSUE THREAD

**Issue by iicdii** [jacklam718/react-native-button-component#13]
I just installed it via npm and run "react-native link" and everything worked fine.
and I tried to render this component but it's not visible on the screen.
I tried to inspect the component by Toggle Inspector on developer menu but I can't touch anything.

## platform
Android, IOS
## package version
"react": "15.4.2",
"react-native": "^0.42.3",
"react-native-button-component": "^0.1.16"
## code
```javascript
import RoundButton from 'react-native-button-component';

...

render() {
  const { err1, err2, orientation } = this.state;

  return (
    <Container style={{ backgroundColor: '#3498db' }}>
      <Content>
        <Grid>
          <Row size={75} style={{ height: orientation === 'PORTRAIT' ? 300 : 130, alignSelf: 'center' }}>
            ...
          </Row>
          <Row size={25}>
            <Content>
              <Form style={{ marginLeft: 10, marginRight: 20 }}>
                ...
                <View>
                  <RoundButton
                    buttonState={this.state.buttonState}
                    backgroundColors={['#212122', '#414141']}
                    text={"Sign In"}
                    onPress={this.handleSubmit.bind(this)}
                  />
                </View>
                ...
              </Form>
            </Content>
          </Row>
        </Grid>
      </Content>
    </Container>
  );
}
```
I also tried to render outside NativeBase Component(Container, Content) but it still doesn't appear.
I tried almost any possible case like removing options, adding width and height, replacing it with other components (ButtonComponent, CircleButton, RectangleButton) but still doesn't work.

**Comment by mswokk** [comment]
Same here. Shows nothing.
I put ButtonComponent between other 2 components, it consumes 0 pixel.

**Comment by jacklam718** [comment]
@iicdii @mswokk  So sorry! I'll investigate it. Thank you for reporting this issue.

**Comment by nishiltamboli** [comment]
+1 to @iicdii 's issue. Same here. Button is not visible at all. Weird.

**Comment by addieljuarez** [comment]
@jacklam718  I have same the problem

**Comment by kaloczidev** [comment]
+1

**Comment by pomeloworkspace** [comment]
+1

**Comment by TGNC** [comment]
+1

**Comment by kayazinc** [comment]
Any updates on this issue?  Thanks

**Comment by jacklam718** [comment]
I have fixed the issue in version `0.1.17`
